packages/frontend/src/lib/table/environment/ColumnStatus.svelte

```diff
@@ -1,8 +1,9 @@
 <script lang="ts">
-import type { EnvironmentState } from '@shared/src/models/IEnvironmentState';
-export let object: EnvironmentState;
+import TasksProgress from '../../progress/TasksProgress.svelte';
+import type { EnvironmentCell } from '/@/pages/environments';
+export let object: EnvironmentCell;
 </script>
 
 <div class="text-sm text-gray-700">
-  {object.status}
+  <TasksProgress mini={true} tasks="{object.tasks}" />
```

Comment:
I would rather have a separate component, responsible of taking the last tasks rather than having a lot of properties to the TasksProgress.

I would rather you only use a TaskElement inside the `Column status` rather than making the TasksProgress too complex

Reply:
Yes, I was wondering what would be the best solution. I'll change to have a separate component